ibm_mq/tests/test_metadata.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2019-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.dev import run_command
+from datadog_checks.ibm_mq import IbmMqCheck
+
+pytestmark = pytest.mark.e2e
+
+
+def test_metadata(instance_metadata, datadog_agent):
+    check = IbmMqCheck('ibm_mq', {}, [instance_metadata])
+    check.check_id = 'test:123'
+    check.check(instance_metadata)
+
+    cmd = ["docker", "exec", "ibm_mq", "dspmqver"]
```

Comment:
Since this cmd is used a couple times, can we make this a constant in `common.py`?